core/selection.js

```diff
@@ -1260,7 +1260,8 @@
 
 		// Editable element might be absent or editor might not be in a wysiwyg mode.
 		var editable = this.editable();
-		return editable && this.mode == 'wysiwyg' ? new CKEDITOR.dom.selection( editable ) : null;
+
+		return editable && this.mode == 'wysiwyg' && editable.editor.status !== 'recreating' ? new CKEDITOR.dom.selection( editable ) : null;
```

Comment:
It would be nice to add some test to really see if `null` is returned when editor is recreating.